.github/workflows/build-spack.yml

```diff
@@ -66,28 +73,35 @@ jobs:
           fetch-depth: ${{format('{0}', matrix.special != 'clang-tidy' && 383 || 0)}}
           fetch-tags: true # to get version information
       - name: Setup Spack
-        uses: spack/setup-spack@0ce61628ed8a32a5664cf01a0e0b5a4834a3b413 # 2024/03
+        uses: spack/setup-spack@5792f7c7055c3707819380b5e3831d2be6e64b6c # 2024/12
         with:
           ref: ${{env.SPACK_REF}}
           buildcache: true
           color: true
           path: spack-src
+      - name: Patch spack
```

Comment:
Should we add an explanation of why these patches are needed (and for 'how long')?

Reply:
Sure, I've added a note (and added a bit more doc to the top of the file about the spack updating). Hopefully in the next week or so the patches will be merged and I can rebuild the stack with a clean version of Spack.